src/workflow-engine/charms/temporal-worker/oci_factory/activities/activity_consume_events.py

```diff
@@ -3,11 +3,21 @@
 
 from oci_factory.activities.consumer.config import Config
 from oci_factory.activities.consumer.schema import SchemaClient
+from oci_factory.notification.mattermost_notifier import (
+    send_message,
+    update_status_and_message,
+)
 
 import logging
 import os
 import subprocess
 
+MM_MESSAGE_TITLE = (
+    "[OCI Factory Temporal Workflow]: Image Rebuild for New Ubuntu Release"
```

Comment:
Can we print some useful information about the activity? today we only have this one but once we have more, it will be useful to trace where the message came from. we have `activity.info()` that provides a lot of metadata about the workflow and activity

Reply:
Yes, we do have a bunch of other information from the `activity.Info` class. Which of them do you think is necessary to report in the MM notification? (FYI, [docs](https://python.temporal.io/temporalio.activity.Info.html).) I would say the only important info for now is the `[workflow_id](https://python.temporal.io/temporalio.activity.Info.html#workflow_id)` and `[workflow_run_id](https://python.temporal.io/temporalio.activity.Info.html#workflow_run_id)`.

Reply:
Y the workflow id and run-id are useful. But also something about the actual activity running this - the activity-id. I'd need to see an example of an output of that object, to be sure of what the info actually looks like, but this is a good start

Reply:
These three added.